app/assets/javascripts/discourse/tests/acceptance/user-bookmark-bulk-actions-test.js

```diff
@@ -0,0 +1,74 @@
+import { click, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import {
+  acceptance,
+  exists,
+  queryAll,
+  updateCurrentUser,
+} from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+import I18n from "discourse-i18n";
+
+acceptance("Bookmark - Bulk Actions", function (needs) {
+  needs.user();
+  needs.pretender((server, helper) => {
+    server.put("/bookmarks/bulk", () => {
+      return helper.response({
+        bookmark_ids: [],
+      });
+    });
+  });
+
+  test("bulk select - options", async function (assert) {
+    updateCurrentUser({ moderator: true });
```

Comment:
Very minor, but you likely don't ned to set moderator to true here. Regular users should be able to update their own bookmarks.